src/switchroot/ostree-mount-util.h

```diff
@@ -134,12 +134,8 @@ read_proc_cmdline_key (const char *key)
 }
 
 static inline char *
-get_aboot_root_slot (void)
+get_aboot_root_slot (const char *slot_suffix)
 {
-  autofree char *slot_suffix = read_proc_cmdline_key ("androidboot.slot_suffix");
```

Comment:
We don't need this recently added EXIT FAILURE now, instead, if androidboot.slot_suffix exists we do the aboot way, or else we do the standard ostree way, or else we return null, which is handled at the caller:

```
ag -A2 -ri get_ostree_target src/switchroot/ostree-system-generator.c src/switchroot/ostree-prepare-root.c src/switchroot/ostree-prepare-root-static.c
src/switchroot/ostree-system-generator.c
66:  autofree char *ostree_target = get_ostree_target ();
67-  if (!ostree_target)
68-    exit (EXIT_SUCCESS);

src/switchroot/ostree-prepare-root.c
115:  autofree char *ostree_target = get_ostree_target ();
116-  if (!ostree_target)
117-    errx (EXIT_FAILURE, "No ostree= cmdline");

src/switchroot/ostree-prepare-root-static.c
119:  autofree char *ostree_target = get_ostree_target ();
120-
121-  if (snprintf (destpath, sizeof (destpath), "%s/%s", root_mountpoint, ostree_target) < 0)
```